github/reactions.go

```diff
@@ -0,0 +1,256 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import "fmt"
+
+// ReactionsService provides access to the reactions-related functions in the
+// GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/reactions/
+type ReactionsService struct {
+	client *Client
+}
+
+// Reaction represents a GitHub reaction.
+type Reaction struct {
+	// ID is the Reaction ID.
+	ID     *int `json:"id,omitempty"`
+	UserID *int `json:"user_id,omitempty"`
+	// Content is the type of reaction.
+	// Possible values are:
+	//     "+1", "-1", "laugh", "confused", "heart", "hooray".
+	Content *string `json:"content,omitempty"`
+}
+
+// ReactionsSummary represents a summary of GitHub reactions.
+type ReactionsSummary struct {
+	TotalCount *int    `json:"total_count,omitempty"`
+	Plus1      *int    `json:"+1,omitempty"`
+	Minus1     *int    `json:"-1,omitempty"`
```

Comment:
Have you considered `PlusOne` and `MinusOne`? Do you think that's more readable or less?

One (possible irrelevant) observation is that had the reaction been named `1+`, then you couldn't name it `1Plus`, but you could name it `OnePlus`. I think this demonstrates that this approach is slightly more scaleable.

Another possibility to consider is to write these as `ThumbsUp` and `ThumbsDown`. The rationale is that `:+1:` has an alias `:thumbsup:`, and `:-1:` has an alias `:thumbsdown:`. Proof: :+1: == :thumbsup:, :-1: == :thumbsdown:`. Also, if you check https://api.github.com/emojis, both variants show up and have equal values.